src/caret_analyze/plot/bokeh/callback_info_interface.py

```diff
@@ -71,61 +80,65 @@ def show(self,
-        if(xaxis_type == 'system_time'):
-            source_df = self._to_dataframe_core('system_time')
-            l1_columns = source_df.columns.get_level_values(1).to_list()
-            fig_args = self._get_fig_args('system time [s]',
-                                          l1_columns[1],
-                                          ywheel_zoom)
-            p = figure(**fig_args)
+        source_df = self._to_dataframe_core(xaxis_type)
+        l1_columns = source_df.columns.get_level_values(1).to_list()
+        fig_args = self._get_fig_args(xaxis_type,
+                                      l1_columns[1],
+                                      ywheel_zoom)
+        p = figure(**fig_args)
+        if xaxis_type == 'system_time':
             apply_x_axis_offset(p, 'x_axis_plot', frame_min, frame_max)
-        elif(xaxis_type == 'sim_time'):
-            source_df = self._to_dataframe_core('sim_time')
-            l1_columns = source_df.columns.get_level_values(1).to_list()
-            fig_args = self._get_fig_args('simulation time [s]',
-                                          l1_columns[1],
-                                          ywheel_zoom)
-            p = figure(**fig_args)
-        elif(xaxis_type == 'index'):
-            source_df = self._to_dataframe_core('index')
-            l1_columns = source_df.columns.get_level_values(1).to_list()
-            fig_args = self._get_fig_args('index',
-                                          l1_columns[1],
-                                          ywheel_zoom)
-            p = figure(**fig_args)
         p.add_tools(Hover)
-        coloring_rule = 'callback'
-        color_selector = ColorSelector.create_instance(coloring_rule)
+
+        # Draw lines
+        color_selector = \
+            ColorSelector.create_instance(coloring_rule='callback')
+        legend_items = []
         for i, callback in enumerate(self._callbacks):
             color = color_selector.get_color(
                 callback.node_name,
                 None,
-                callback.callback_name)
+                callback.callback_name
+            )
             line_source = get_callback_lines(callback,
                                              source_df,
                                              l1_columns,
                                              frame_min,
                                              xaxis_type)
-            p.line('x',
-                   'y',
-                   source=line_source,
-                   legend_label=f'callback{i}',
-                   color=color)
-        p.add_layout(p.legend[0], 'right')
+            legend_label = f'callback{i}'
+            renderer = p.line('x',
+                              'y',
+                              source=line_source,
+                              color=color)
+            legend_items.append((legend_label, [renderer]))
+
+        # Add legends
+        num_legend_threshold = 20
```

Comment:
20という閾値の根拠もコメントで追加していただきたいです
```suggestion
        num_legend_threshold = 20
        # In Autoware, the number of callbacks in a node is less than 20.
        # Here, num_legend_threshold is set to 20 as the maximum value.
```